phpcs.xml

```diff
@@ -11,9 +11,9 @@
 		<exclude name="Squiz.Commenting.FileComment.SpacingAfterComment"/>
 	</rule>
 	
-	<!--For Windows users that may have problems with EOL-->
-	<!-- <rule ref="Generic.Files.LineEndings">
+	<!--For Windows users that may have problems with EOL. If you are not using windows please coment the code bellow-->
```

Comment:
```suggestion
	<!-- For Windows users that may have problems with EOL. If you are not using Windows please comment out the code below -->
```

Reply:
I did both changes (comment sugestion and variable name changing)